pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -0,0 +1,208 @@
+package org.broadinstitute.dsm.util;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+import static org.broadinstitute.dsm.statics.ESObjectConstants.ONC_HISTORY_DETAIL;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.model.elastic.Profile;
+import org.broadinstitute.dsm.model.elastic.export.painless.UpsertPainless;
+import org.broadinstitute.dsm.util.export.ElasticSearchParticipantExporterFactory;
+import org.broadinstitute.dsm.util.export.ParticipantExportPayload;
+import org.broadinstitute.lddp.handlers.util.Institution;
+import org.broadinstitute.lddp.handlers.util.InstitutionRequest;
+import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
+import org.elasticsearch.action.get.GetRequest;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.indices.CreateIndexRequest;
+import org.elasticsearch.client.indices.CreateIndexResponse;
+import org.elasticsearch.client.indices.PutMappingRequest;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.xcontent.XContentType;
+import org.junit.Assert;
+
+@Slf4j
+public class ElasticTestUtil {
+
+    public static String createIndexWithMappings(String realm, String mappingsFile) {
+        String indexName = createIndex(realm);
+        try {
+            String mappingsJson = TestUtil.readFile(mappingsFile);
```

Comment:
@ssettipalli what can we do here to ensure that when we redefine indexes during deployments, we make the appropriate changes here to our tests so the index structure doesn't drift from reality?